ramalama/hf_style_repo_base.py

```diff
@@ -336,6 +337,8 @@ def _pull_with_model_store(self, args):
             self.store.new_snapshot(tag, snapshot_hash, files)
 
         except Exception as e:
+            if not attempt_cli_fallback:
```

Comment:
I would prefer this to not be a begative.

skip_fallback = "/" not in organization

If skip_fallback:
      raise